packages/eslint-plugin-template/tests/rules/no-call-expression.test.ts

```diff
@@ -31,45 +31,46 @@ ruleTester.run(RULE_NAME, rule, {
   ],
   invalid: [
     convertAnnotatedSourceToFailureCase({
-      description: 'it should fail with call expression in expression binding',
+      description: 'it should fail for call expression in a expression binding',
       annotatedSource: `
         <div>{{ getInfo() }}</div>
                 ~~~~~~~~~
       `,
       messageId,
     }),
-    convertAnnotatedSourceToFailureCase({
-      description: 'it should fail with call expression in expression binding',
-      annotatedSource: `
-        <a href="http://example.com">{{ getInfo() }}</a>
-                                        ~~~~~~~~~
-      `,
-      messageId,
-    }),
     convertAnnotatedSourceToFailureCase({
       description:
-        'it should fail when using a property resulted from a call expression in expression binding',
+        'it should fail when using a property resulted from a call expression in a expression binding',
       annotatedSource: `
         <a href="http://example.com">{{ getInfo().name }}</a>
                                         ~~~~~~~~~
       `,
       messageId,
     }),
     convertAnnotatedSourceToFailureCase({
-      description: 'it should fail with call expression in property binding',
+      description: 'it should fail for call expression in a property binding',
       annotatedSource: `
         <a [href]="getUrl()">info</a>
                    ~~~~~~~~
       `,
       messageId,
     }),
     convertAnnotatedSourceToFailureCase({
-      description: 'it should fail with a property access call expression',
+      description: 'it should fail for safe/unsafe method calls',
       annotatedSource: `
-        <a [href]="helper.getUrl()">info</a>
-                   ~~~~~~~~~~~~~~~
+        {{ obj?.nested1() }} {{ obj!.nested1() }}
+            ~~~~~~~~~~~~~~       ^^^^^^^^^^^^^^      
```

Comment:
@JamesHenry, 2 points to notice here:

- While I was doing this, I remembered that we discussed this at https://github.com/angular-eslint/angular-eslint/pull/205#discussion_r532132323, so I decided to debug deeper this time and finally figured out why it happens. The "problem" is that Angular condenses consequent interpolations in a single `BoundText`, and so in order to check the error by passing a `char`, we should inline all the interpolations in a single line.

- Despite having solved this, I noticed another one where the error column is off by one (the same happens in the `template-no-any`)... maybe is it an bug with `getLocFromIndex`? I also noticed that it was reported in https://github.com/mgechev/codelyzer/issues/859.